packaging/app.organicmaps.desktop.metainfo.xml

```diff
@@ -113,6 +113,29 @@
   </screenshots>
 
   <releases>
+   <release version="2023.04.02-7" date="2023-04-02">
+     <description>
+       <p>Highlights:</p>
+         <ul>
+           <li>New OpenStreetMap data as of March 29</li>
+           <li></li>
+           <li></li>
+         </ul>
+       <p>Map Styles:</p>
+         <ul>
+           <li>Many new shop types</li>
```

Comment:
• New OpenStreetMaps data as of March 29
• Correctly import KML bookmarks with track data
• Allow adding music and musical_instruments in the Editor
• Sewing shops and haberdasheries are searchable
• Distinguish supermarkets and convenience stores
• New shop types: agrarian, appliance, bed, outpost, gas, key cutter, locksmith, carpet, motorcycle repair, pet grooming, hi-fi, bathroom furnishing, curtain, fashion accessories, cannabis, herbalist
• Added money transfer, bicycle charging stations, power plant
• Made captions mandatory for capitals with a star icon
• Made park, forest, and scrub colors different
• Added metal construction icons
• Improved aerial ways, cable cars, power lines, fences
• Reduce Barcelona subway visibility
• Fixed isolines that are drawn over buildings, roads, etc.
• Fixed some POI captions displayed earlier than their icons
• Improved German, Turkish, Belarusian, Hungarian translations
• Changed displayed name to Organic Maps
• OM version is selectable in About

Reply:
Done